packages/ag-charts-community/src/util/keynavUtil.ts

```diff
@@ -8,6 +8,20 @@ function addRemovableEventListener<K extends keyof HTMLElementEventMap>(
     destroyFns.push(() => button.removeEventListener(type, listener));
 }
 
+function addEscapeEventListener(
+    destroyFns: (() => void)[],
+    elem: HTMLElement,
+    onEscape?: (event: KeyboardEvent) => void
+) {
+    if (onEscape) {
```

Comment:
I don't think we should be pusing `undefined` handling down like this - there is no point invoking this method if `onEscape` is undefined in the first place, and makes the calling code less readable because it looks like an event handler is always added?

Reply:
ACK.